charts/opentelemetry-collector/values.schema.json

```diff
@@ -257,6 +263,9 @@
         }
       }
     },
+    "lifecycleHooks": {
```

Comment:
Let's restrict this to only allowing the `PostStart` and `PreStop` properties.

Reply:
I added `exec` and `httpGet` type handlers (only one can be active) with their possible properties, as defined here: https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#LifecycleHandler